test/Microsoft.AspNetCore.Server.WebListener.FunctionalTests/ResponseHeaderTests.cs

```diff
@@ -133,7 +133,8 @@ public class ResponseHeaderTests
                 response.EnsureSuccessStatusCode();
                 Assert.True(response.Headers.ConnectionClose.Value);
                 Assert.Equal(new string[] { "close" }, response.Headers.GetValues("Connection"));
-                Assert.False(response.Headers.TransferEncodingChunked.HasValue);
+                Assert.True(response.Headers.TransferEncodingChunked.HasValue);
+                Assert.True(response.Headers.TransferEncodingChunked);
```

Comment:
Cool. Might want to put Http11 in the test name to be clear the Http10 behavior is different for this test case.